pkg/rbd/controllerserver.go

```diff
@@ -24,6 +24,7 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/ceph/ceph-csi/pkg/util"
```

Comment:
where is the pkg?

Reply:
@rootfs : @mickymiek has added a new package under `pkg/util`, that's one of the reasons the CI is failing but should be solved once we get this merged